src/nodes/snat_node.c

```diff
@@ -22,37 +23,41 @@ static __rte_always_inline rte_edge_t get_next_index(__rte_unused struct rte_nod
 {
 	struct dp_flow *df = dp_get_flow_ptr(m);
 	struct flow_value *cntrack = df->conntrack;
+	struct snat_data *snat_data_p = NULL;
```

Comment:
I've only ever seen `*_p` used for double-pointers in this codebase. I think this should stay `snat_data` and the other one can be `snat64_data`